test/functional/apps/home/index.js

```diff
@@ -29,6 +29,7 @@ export default function ({ getService, loadTestFile }) {
 
     loadTestFile(require.resolve('./_navigation'));
     loadTestFile(require.resolve('./_home'));
+    loadTestFile(require.resolve('./_newsfeed'));
```

Comment:
This way tests will be executed under default config

Reply:
it is supposed to be execute this way, because in this file is only success newsfeed tests